brim/release

```diff
@@ -1,45 +1,64 @@
 #!/bin/sh -ex
 
-case $(uname -s) in
+install_libpcap() {
```

Comment:
can you add the comment stating why we do this from the removed text ("Compile a recent libpcap, ...")

Reply:
@alfred-landrum: That comment is still present at https://github.com/brimsec/zeek/blob/windows/brim/release#L22-L23.